arch/src/main/java/arch/extensions/navExtensions.kt

```diff
@@ -20,6 +24,25 @@ fun NavController.navigate(navEvent: NavigationEvent) {
     }
 }
 
+fun NavController.safeNavigate(@IdRes resId: Int, args: Bundle? = null, navOptions: NavOptions? = null) {
```

Comment:
Moc se mi nelíbí ten naming safe - neměly by být všechny navigate cally safe? Kdy se má volat unsafe?

Reply:
Záměr je to tímto způsobem odlišit od funkcí `navigate(..)` přímo v `NavController` aby nedošlo k importování špatné funkce.

Reply:
ok